recipe/meta.yaml

```diff
@@ -35,14 +37,23 @@ test:
     - kombu.transport
     - kombu.transport.virtual
     - kombu.utils
+  requires:
+    - pip
+  commands:
```

Comment:
There are some unit tests present. Do we want to add them?

Reply:
I've added a test suite.